Sources/SpeziAccount/SpeziAccount.docc/Account Values/Adding new Account Values.md

```diff
@@ -97,6 +97,26 @@ Therefore, you typically do not need to provide a custom view implementation,
 or you might consider adding `CustomLocalizedStringResourceConvertible` protocol
 conformance to your `Value` type.
 
+However, if your `Value` is not `String`-based, you need to implement your own DataDisplayView. Below is a short code example that demonstrates how to implement a DataDisplayView for which the `Value` is `Int`-based for a HeightValue Key.
+```swift
+public struct DataDisplay: DataDisplayView {
+    public typealias Key = HeightKey
+    private let height: Int
+    public init(_ value: Int) {
+        self.height = value
+    }
+    public var body: some View {
+        HStack {
+            Text(HeightKey.name)
+            Spacer()
+            Text("\(height) cm")
+                .foregroundColor(.secondary)
+        }
+        .accessibilityElement(children: .combine)
+    }
+}
+```
```

Comment:
I have a few points I would like to see improved here in term of the code style of the code example itself.

Please, place the `DataDisplay` view within an extension of the (example) `HeightKey` type so it is more in line with how we propose to structure AccountKey implementations and where this code example belongs to. So the code example might then look like the following:
```swift
extension HeightKey {
    public struct DataDisplay: DataDisplayView {
        // ...
    }
}
```